### ISSUE TITLE
Problem with G5K template

### ISSUE THREAD

**Issue by Shurakai** [oar-team/kameleon#67]
When executing
  kameleon new debian-stretch-g5k default/grid5000/debian8

I get 

Error : Template 'default/grid5000/debian8.yaml' not found. To see all templates, run the command `kameleon template list`

I copy-pasted the "default/grid5000/debian8" from the output of "template liste" and did not change any directory or executed any command in between.

Executing "default/chroot/debian8" works fine, though.

**Comment by npf** [comment]
Hello, Christian,

Did you start from scratch with kameleon, or looked at some point to the Grid'5000 receipes at
https://github.com/grid5000/environments-recipes ?

**Comment by Shurakai** [comment]
Hi,

I installed kameleon as described on the website (installed via gem, checked out the main template repository). There is a file that declares the default/grid5000/debian8 template (I checked), but kameleon doesn't find it for whatever reason...

**Comment by npf** [comment]
Well ok, sorry I might have misunderstood what you want to do.

If you want to use kameleon using Grid'5000 as an execution backend, you are in the right place here.

The link I gave is about generating kadeploy environnements for Grid'5000, which is slightly different.

Assigning to @mickours

**Comment by SalemHarrache** [comment]
Hi Chrisitian,

I've just seen this bug and I confirm it appeared with the latest releases.

I've fixed it (it should work fine now) and I've published the bugfix release 2.7.1

Don't forget to mention the variables, in the recipe or in command line, like this:

```
$ kameleon build debian-stretch-g5k.yaml -g g5k_site:lyon g5k_user:cheinrich
```

Thanks for the report !
